docs/src/reference/experimental/loop-contracts.md

```diff
@@ -116,29 +119,75 @@ must be sufficient to establish subsequent assertions. If it is not, the abstrac
 
 To illustrate the key idea, we show how Kani abstracts the loop in `simple_loop_with_loop_contracts` as a non-loop block:
 ``` Rust
-assert!(x >= 1) // check loop invariant for the base case.
+assert!(x >= 1) // check loop contract for the base case.
 x = kani::any();
 kani::assume(x >= 1);
 if x > 1 {
     // proof path 1:
-    //   both loop guard and loop invariant are satisfied.
+    //   both loop guard and loop contract are satisfied.
     x = x - 1;
-    assert!(x >= 1); // check that loop invariant is inductive.
+    assert!(x >= 1); // check that loop contract is inductive.
     kani::assume(false) // block this proof path.
 }
 // proof path 2:
-//   loop invariant is satisfied and loop guard is violated.
+//   loop contract is satisfied and loop guard is violated.
 assert!(x == 1);
 ```
-That is, we assume that we are in an arbitrary iteration after checking that the loop invariant holds for the base case. With the inductive hypothesis (`kani::assume(x >= 1);`),
+That is, we assume that we are in an arbitrary iteration after checking that the loop contract holds for the base case. With the inductive hypothesis (`kani::assume(x >= 1);`),
 we will either enter the loop (proof path 1) or leave the loop (proof path 2). We prove the two paths separately by killing path 1 with `kani::assume(false);`.
 Note that all assertions after `kani::assume(false)` will be ignored as `false => p` can be deduced as `true` for any `p`.
 
-In proof path 1, we prove properties inside the loop and at last check that the loop invariant is inductive.
+In proof path 1, we prove properties inside the loop and at last check that the loop contract is inductive.
```

Comment:
I think these specific changes from `invariant` to `contract` should also be reverted.

Reply:
For me, loops have contracts that come in many forms: modifies or "assigns", invariants, "variant" or "decreases" contracts and so on.

A "loop invariant" is one type of "loop contract" which has a specific meaning and use in verification.

So... say "loop contract" when you mean any one of them. Say "loop invariant" when you specifically mean the thing that really is the invariant of the loop.

This conversation should not be necessary in 2025!